code/atelier style.css

```diff
@@ -0,0 +1,203 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+body {
+background: whitesmoke;
+color: black;
+margin: 0;
+font-family: 'Montserrat', sans-serif;
+}
```

Comment:
Indentation:
```
body {
  background: whitesmoke;
  color: black;
  margin: 0;
  font-family: 'Montserrat', sans-serif;
}
```

Take a look at our [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) to learn more.